### ISSUE TITLE
NVDA Android studio editor error is not spoken

### ISSUE THREAD

**Issue by Adriorjalesvidal** [nvaccess/nvda#12216]
### Steps to reproduce:
1. Create an Android Studio project (tested with Java languaje selected)
2. Make a mistake on the code editor, like deleting the semicolon at the end of a line. 
3. Press F2 to navigate to the next error on the file
4. Press CTRL + F1 to read the error message
5. NVDA does not read any message. 

### Actual behavior:
NVDA doesn't read the code editor error description on Android Studio

### Expected behavior:
NVDA must read the description after pressing CTRL+F1 or after moved to the next/previous error using the F2/shift+F2 keystrokes

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
NVDA 2020.4

#### Windows version:
Windows 10 20h2

#### Name and version of other software in use when reproducing the issue:
Android studio 4.1.2

#### Does the issue still occur after restarting your computer?
Yes

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No

#### If add-ons are disabled, is your problem still occurring?
Yes

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
Yes, and issue still occurring

**Comment by akash07k** [comment]
Yes, even this bug is there with latest android studio caneri too.
would be good if it gets fixed.

**Comment by Adriorjalesvidal** [comment]
Latest working release of android studio is release 3.6.3

**Comment by akash07k** [comment]
Sorry buddy, I had to right latest Android Studio Caneri 4.1.3

**Comment by cary-rowen** [comment]
This problem has not been solved yet.

@TheQuinbox has developed an app module that you can get in the add-on store.

**Comment by akash07k** [comment]
@cary-rowen  any performance concerns with that addon?

**Comment by mwhapples** [comment]
With AndroidStudio issues, I would always suggest also testing with the latest IntelliJ, AndroidStudio trails quite a bit on the IDE side of things being based off an old version of IntelliJ.

**Comment by cary-rowen** [comment]
@akash07k 
> any performance concerns with that addon?

The only thing to worry about might be the implementation for reporting the status bar, which might be a little time consuming, but it's bearable.
Overall, I don't think there are performance worries.


Hi @mwhapples 
Yes, I also tested with IntelliJ IDEA and it still looks like I can reproduce this.

IntelliJ IDEA 2023.3.2 (Community Edition)
Build #IC-233.13135.103, built on December 20, 2023
Runtime version: 17.0.9+7-b1087.9 amd64
VM: OpenJDK 64-Bit Server VM by JetBrains s.r.o.
Windows 10.0
